options.go

```diff
@@ -111,6 +112,7 @@ func DefaultOptions(path string) Options {
 		Truncate:           false,
 		Logger:             defaultLogger,
 		LogRotatesToFlush:  2,
+		StorageKeyPath:     "",
```

Comment:
No need for this, the default value is already an empty string.
You could even remove the `Truncate` actually